migrations/README.rst

```diff
@@ -168,7 +168,24 @@ For doing that the following command might be used:
 
     alembic downgrade head REVISION_ID
 
-| If you would like to use relative identifiers, such as :code:`alembic downgrade head -1`, you might check it out: `Relative Identifiers <https://alembic.sqlalchemy.org/en/latest/tutorial.html#relative-migration-identifiers>`_
+| If you would like to use relative identifiers, such as :code:`alembic downgrade -1`, you might check it out: `Relative Identifiers <https://alembic.sqlalchemy.org/en/latest/tutorial.html#relative-migration-identifiers>`_
+|
+| **Note-4:** During the migration of SQLite Database, it's possible to see this error:
+
+.. code-block:: bash
+
+    AddGeometryColumn() error: "UNIQUE constraint failed: geometry_columns.f_table_name, geometry_columns.f_geometry_column"
+
+You can ignore this error because it says that there is already a geometry column entity for your table that has a Geometry column (i.e. States, Geometries, Contact). This error happens because SQLite doesn't support *ALTER TABLE* statement.
```

Comment:
If we see this error, does the rest of the migration continue correctly? I'd have thought it'd have stopped part-way through, potentially leaving the database in an unclean state?

Reply:
Yes, it doesn't affect the `geometry` column because it means that there is already a column (which is created in the previous migrations). If you migrate your database in this PR, you will see that this error is thrown for a few migrations but it doesn't change anything. 

I know that it sounds inconvenient. When I searched for it, I found out that geometry columns accept `management` parameter. If it's true, it should automatically drop this value: (https://geoalchemy-2.readthedocs.io/en/latest/spatialite_tutorial.html#declare-a-mapping)

`Setting management to True indicates that the AddGeometryColumn and DiscardGeometryColumn management functions will be used for the creation and removal of the geometry column. This is required with SpatiaLite.`

In our columns, we already set this parameter to `True` but Alembic still doesn't drop it when a table with geometry column is dropped. I don't have any idea why it's happening.

Reply:
Ok, that's fair enough.